CRM/Activity/Page/AJAX.php

```diff
@@ -56,7 +56,7 @@ public static function getCaseGlobalRelationships() {
     // get the total row count
     CRM_Case_BAO_Case::getGlobalContacts($globalGroupInfo, NULL, FALSE, TRUE, NULL, NULL);
     // limit the rows
-    $relGlobal = CRM_Case_BAO_Case::getGlobalContacts($globalGroupInfo, $params['sortBy'], $showLinks = TRUE, FALSE, $params['offset'], $params['rp']);
+    $relGlobal = CRM_Case_BAO_Case::getGlobalContacts($globalGroupInfo, CRM_Utils_Array::value('sortBy', $params), $showLinks = TRUE, FALSE, $params['offset'], $params['rp']);
```

Comment:
Thanks for the PR - I've noticed these messages too. FYI @colemanw might revoke your Civi card if he sees this line. Use `$params['sortBy'] ?? NULL`

Reply:
Thanks @demeritcowboy. Have updated the PR now.